src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -1,22 +1,40 @@
 import _ from 'lodash/fp';
 import React from 'react';
 import fullSchema from 'vets-json-schema/dist/complaint-tool-schema.json';
-import FormFooter from '../../../../platform/forms/components/FormFooter';
-import fullNameUI from '../../../../platform/forms/definitions/fullName';
 import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
 import phoneUI from 'us-forms-system/lib/js/definitions/phone';
 import { validateBooleanGroup } from 'us-forms-system/lib/js/validation';
 
+import FormFooter from '../../../../platform/forms/components/FormFooter';
+import fullNameUI from '../../../../platform/forms/definitions/fullName';
+import dataUtils from '../../../../platform/utilities/data/index';
+
+const { get, omit, set } = dataUtils;
+
 import IntroductionPage from '../containers/IntroductionPage';
 import ConfirmationPage from '../containers/ConfirmationPage';
 import SchoolSelectField from '../../components/SchoolSelectField.jsx';
 import GetFormHelp from '../../components/GetFormHelp';
 
 import { transform, submit } from '../helpers';
 
-const { educationDetails } = fullSchema.properties;
+const {
+  address: applicantAddress,
+  anonymousEmail,
+  applicantEmail,
+  educationDetails,
+  fullName,
+  issue,
+  issueDescription,
+  issueResolution,
+  onBehalfOf,
+  phone,
+  serviceAffiliation,
+  serviceBranch,
+  serviceDateRange,
+} = fullSchema.properties;
 
-const { school } = educationDetails;
+const { assistance, programs, school } = educationDetails;
 
 const { schoolInformation } = school.oneOf[0];
```

Comment:
This will be replaced with `const schoolInformation = school.oneOf[0];` once the updated BE schema is used.